### PR TITLE
Update git ignore

### PR DESCRIPTION
**Pull request by supereagle**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,11 +13,5 @@ bin
 # Dirty file in OS X
 .DS_Store
 
-# Config when used with VS Code
-.vscode/
```
Comment (cd1989) [inline .gitignore]: Why ?

Comment (supereagle) [reply .gitignore]: Global configuration can avoid `.vscode` for each project.

Comment (cd1989) [reply .gitignore]: But we can't assume everyone have this global configuration. And any problem if we configure it again ?

Comment (zhujian7) [reply .gitignore]: Agree with cd1989

**Comment by supereagle** [comment]
Have fixed comments.

**Pull request merged**